src/sass/vic-v2.scss

```diff
@@ -287,6 +281,19 @@ input[type=range] {
   }
 }
 
+.narrow-layout {
+  @media (min-width: $large-screen) {
+    display: none;
+  }
+}
+
+.normal-layout {
```

Comment:
Using these for determining if elements should be visible.

Reply:
These names make sense. Also it's good to know that @media works this way, rather than according to window width.